infoschema/cluster_tables_test.go

```diff
@@ -415,20 +415,30 @@ func TestStmtSummaryHistoryTableWithUserTimezone(t *testing.T) {
 	tk.MustExec("use test;")
 	tk.MustExec("set time_zone = '+08:00';")
 	tk.MustExec("select sleep(0.1);")
-	r := tk.MustQuery("select FIRST_SEEN, LAST_SEEN from INFORMATION_SCHEMA.STATEMENTS_SUMMARY_HISTORY order by LAST_SEEN limit 1;")
+	r := tk.MustQuery("select FIRST_SEEN, LAST_SEEN, SUMMARY_BEGIN_TIME, SUMMARY_END_TIME from INFORMATION_SCHEMA.STATEMENTS_SUMMARY_HISTORY order by LAST_SEEN limit 1;")
 	date8First, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][0].(string))
 	require.NoError(t, err)
 	date8Last, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][1].(string))
 	require.NoError(t, err)
+	date8Begin, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][2].(string))
+	require.NoError(t, err)
+	date8End, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][3].(string))
+	require.NoError(t, err)
 	tk.MustExec("set time_zone = '+01:00';")
-	r = tk.MustQuery("select FIRST_SEEN, LAST_SEEN from INFORMATION_SCHEMA.STATEMENTS_SUMMARY_HISTORY order by LAST_SEEN limit 1;")
+	r = tk.MustQuery("select FIRST_SEEN, LAST_SEEN, SUMMARY_BEGIN_TIME, SUMMARY_END_TIME from INFORMATION_SCHEMA.STATEMENTS_SUMMARY_HISTORY order by LAST_SEEN limit 1;")
 	date1First, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][0].(string))
 	require.NoError(t, err)
 	date1Last, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][1].(string))
 	require.NoError(t, err)
+	date1Begin, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][2].(string))
+	require.NoError(t, err)
+	date1End, err := time.Parse("2006-01-02 15:04:05", r.Rows()[0][3].(string))
+	require.NoError(t, err)
 
 	require.Less(t, date1First.Unix(), date8First.Unix())
 	require.Less(t, date1Last.Unix(), date8Last.Unix())
+	require.Less(t, date1Begin.Unix(), date8Begin.Unix())
+	require.Less(t, date1End.Unix(), date8End.Unix())
```

Comment:
It should be `date1Begin=date8Begin` and `date1End=date8End` because the statements summary refreshing is disabled.
Besides, I don't think this is related to the issue to be closed.

Reply:
nope, that is the feature they want